Sources/MessagingInApp/Gist/Managers/MessageQueueManager.swift

```diff
@@ -3,7 +3,8 @@ import Foundation
 import UIKit
 
 protocol MessageQueueManager: AutoMockable {
-    var interval: Double { get set }
```

Comment:
Although I do prefer to use property getter/setters for an API, I found this API to be safer with having the class be a singleton. 

Here is some sample code with what I mean: 

```swift
// If you use a property getter/setter API:
let queueManager: MessageQueueManager = ... get singleton instance 
queueManager.interval = 500
// 🤔 Did we update the shared singleton `inventory` property? Or did we update the `queueManager` local variable, only? 

// if we change to getter/setter functions:
MessageQueueManager.singletonInstance.setInventory(500)
// This is much more clear that we are editing the singleton, not a locally scoped instance. 
```